app/views/users/show.html.erb

```diff
@@ -6,8 +6,8 @@
     <%= @user.bio %>
     </p>
   </div>
-  <div>
-    <%= render @user.most_recent_posts, show_comments: false %>     
+  <div id="posts-container">
+    <%= render @user.most_recent_posts.includes(:author), show_comments: false %>     
```

Comment:
- [ ] Please make sure to fix the N+1 problem in the controllers and not in the views so that we can eliminate N+1 queries happening in the views.

Reply:
Hi, @ShadyShawkat Thank you so much for your review! Do you happen to have any references on this topic that presents the difference? I would like to know better.

In this case, I'm using the `most_recent_posts` method which is in my User model, should I add the includes there?